ingress/controllers/nginx-third-party/main.go

```diff
@@ -62,14 +62,24 @@ func main() {
 		glog.Fatalf("Please specify --default-backend")
 	}
 
+	glog.Info("Checking if DNS is working")
+	ip, err := checkDNS(*defaultSvc)
```

Comment:
> or am I missing something

nginx is not using the IP of the services and without a valid server to resolve the addresses it will fail to return content from the services